Backend/README.md

```diff
@@ -0,0 +1,25 @@
+# iDO Backend
+
+## What is this?
+
+This is the node.js express server that will provide the backend API for our react native frontend application. It is connected to an [mLab](https://mlab.com/) mongodb databse that is hosted in AWS. 
+
+## Development
+
+### Setup
+
+Download [keepassX](https://www.keepassx.org/downloads) (You may have to allow for third-party applications in your settings)
+
+Open `secrets.kdbx` and follow the instructions in the file `mlab db ido-development` under the development group.
+
+
+### Run
+
+```bash
+$ npm install
+$ npm start
+
+> ido.backend@0.0.1 start <PATH>
```

Comment:
I wouldn't stick lines 22-25 in the readme - it's not very useful info.
Instead, add a line to explain that the server will be running on port x

Reply:
Yeah, fair point, I didn't have it originally but decided to put it in last minute. Happy to remove.